src/util/tracking.js

```diff
@@ -172,7 +172,7 @@ export const logTrack = platformAgent => (response, trackInfo) => {
 	const eventDate = new Date();
 	const record = {
 		timestamp: eventDate.getTime().toString(),
-		requestId: uuid.v4(),
+		requestId: response.request.id,
```

Comment:
Ah, I see... so this means that both click/activity tracking share a UUID right?